e2e/test/scenarios/native-filters/helpers/e2e-field-filter-helpers.js

```diff
@@ -109,9 +109,12 @@ export function addWidgetNumberFilter(
  * @return {function}
  */
 export function addDefaultNumberFilter(value) {
-  return isBetweenFilter(value)
-    ? addBetweenFilter(value)
-    : enterDefaultValue(value);
+  if (isBetweenFilter(value)) {
+    cy.findByText("Enter a default value…").click();
```

Comment:
`EnterDefaultFilter` has it's own click to open the popover, but `addBetweenFilter` does not (and is used to populate dashboad filters on other screens). This just ensures that the default popover value is open before we get to `addBetweenFilter`